api/python/tests/test_cli.py

```diff
@@ -88,12 +129,15 @@ def test_push_with_meta_data_error(
     with patch_package_class as mocked_package_class, \
          mock.patch('quilt3.main.parse_arg_json', wraps=main.parse_arg_json) as mocked_parse_json_arg:
```

Comment:
Not for this PR, but there's an extra leading space here. IDK why pylint doesn't complain.

Reply:
nm, i'm wrong